test/CarouselSpec.js

```diff
@@ -287,4 +287,24 @@ describe('<Carousel>', () => {
       clock.restore();
     }
   });
+  it('should handle touch events', () => {
+    const clock = sinon.useFakeTimers();
+
+    try {
+      const onSelectSpy = sinon.spy();
+      const wrapper = mount(
+        <Carousel interval={0} onSelect={onSelectSpy} touch>
+          {items}
+        </Carousel>,
+      );
+
+      wrapper.simulate('touchStart', { changedTouches: [{ screenX: 0 }] });
```

Comment:
Hmm, can't seem to figure out why this isn't firing the touch event handler...

Reply:
Are you manually attaching touch handlers or using the react event system? Enzyme will simulate at the react level, but not actually fire Dom events